subprojects/core/src/main/java/org/gradle/api/internal/changedetection/state/AbiExtractingClasspathResourceHasher.java

```diff
@@ -47,7 +47,9 @@ private HashCode hashClassBytes(InputStream inputStream) throws IOException {
         if (extractor.shouldExtractApiClassFrom(reader)) {
             byte[] signature = extractor.extractApiClassFrom(reader);
             if (signature != null) {
-                return Hashing.md5().hashBytes(signature);
+                Hasher hasher = Hashing.newHasher();
```

Comment:
Why don't we use `Hashing.hashBytes` here? I don't see any reason to use `newHasher`.

Reply:
Changed.